src/hotspot/cpu/s390/s390.ad

```diff
@@ -6253,6 +6253,29 @@ instruct divI_reg_imm16(roddRegI dst, iRegI src1, immI16 src2, revenRegI tmp, fl
   ins_pipe(pipe_class_dummy);
 %}
 
+// Unsigned Integer Register Division
+// NOTE: z_dlr requires even-odd pair. remainder will be in even register(r4) & quotient will be stored in odd register(r5)
+// for dividend, upper 32bits will be in r4 and lower 32bits will be in r5 register.
+instruct udivI_reg_reg(roddRegI r5_rodd_dst, iRegI src2, revenRegI r4_reven_tmp, flagsReg cr) %{
+  match(Set r5_rodd_dst (UDivI r5_rodd_dst src2));
+  effect(TEMP r4_reven_tmp, KILL cr);
+  // TODO: size(4);
+  format %{ "UDIV $r5_rodd_dst, $r5_rodd_dst,$src2" %}
```

Comment:
Suggestion: no whitespace between instruction operands.

Reply:
removed

Reply:
Hi @RealLucy, 
was that for consistency or there is issue with space in the format section?